packages/dashboard-provider/manual-test/metacoin-hardhat/contracts/MetaCoin.sol

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: MIT
+pragma experimental ABIEncoderV2;
+pragma solidity >=0.4.25 <0.9.0;
+
+import "./ConvertLib.sol";
+
+// This is just a simple example of a coin-like contract.
+// It is not standards compatible and cannot be expected to talk to other
+// coin/token contracts. If you want to create a standards-compliant
+// token, see: https://github.com/ConsenSys/Tokens. Cheers!
+
+contract MetaCoin {
```

Comment:
the truffle repo is a hungry beast that demands we continue to define `MetaCoin.sol` in new places all the time.

praise be the demands of the code

(sorry it's getting late here)